CONTRIBUTING.md

```diff
@@ -1,26 +1,26 @@
 ## Contributing to the C++ Core Guidelines
 
->"Within C++ is a smaller, simpler, safer language struggling to get out." 
->-- <cite>Bjarne Stroustrup</cite>
+>“Within C++ is a smaller, simpler, safer language struggling to get out.”<br />
```

Comment:
@jwakely Originally I tried to use `\` at the end of line (it should create a line break in Markdown), but it didn’t work on GitHub for some reason. Could you provide a working example of adding a line break to an *.md file with no HTML?

P.S. Looks like different Markdown parsers are used for GitHub comments and *.md files:
in comments, line break is added automatically just based on new-line character while in *.md files new line character is ignored.